test/test_torch.py

```diff
@@ -4579,6 +4581,9 @@ def test_copy_broadcast(self):
         torch.zeros(5, 6).copy_(torch.zeros(6))
         self.assertRaises(RuntimeError, lambda: torch.zeros(5, 6).copy_(torch.zeros(30)))
 
+    def test_copy_many_to_one(self):
+        self.assertRaises(RuntimeError, lambda: torch.zeros(1, 6).expand(5, 6).copy_(torch.zeros(5, 6)))
```

Comment:
super nit: such one lines makes it hard to understand what is supposed to raise an exception here

Reply:
Agreed.
Will comment for the test case.